src/Amp/Database/Datasource.php

```diff
@@ -175,6 +175,16 @@ function updateHost() {
     }
   }
 
+  /**
+   * Convert a PDO driver name to a PHP extension name.
+   */
+  function convertPDODrivertoPHPExtension($driver) {
+    if ($driver == 'mysql') {
```

Comment:
wouldn't that also imply that there has to be a check in e.g. \Amp\Command\CreateCommand to see if mysql of mysqli is enabled? As Amp should be able to work with both (with preference on mysqli)?